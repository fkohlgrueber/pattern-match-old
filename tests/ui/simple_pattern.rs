

fn main() {
    
    // match literal false
    if (false) {
    }

    // match array sequences
    let a = ['b', 'c'];
    let b = ['b', 'b', 'c'];
    let no_warn = ['b'];
    
    // match cast
    let x = 0 as *const i32;    
}
