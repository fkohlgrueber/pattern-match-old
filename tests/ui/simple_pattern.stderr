error: This is a match for a simple pattern. Well Done!
 --> $DIR/simple_pattern.rs:6:9
  |
6 |     if (false) {
  |         ^^^^^
  |
  = note: #[forbid(simple_pattern)] on by default

error: This is a match for a simple pattern. Well Done!
  --> $DIR/simple_pattern.rs:10:13
   |
10 |     let a = ['b', 'c'];
   |             ^^^^^^^^^^

error: This is a match for a simple pattern. Well Done!
  --> $DIR/simple_pattern.rs:11:13
   |
11 |     let b = ['b', 'b', 'c'];
   |             ^^^^^^^^^^^^^^^

error: This is a match for a simple pattern. Well Done!
  --> $DIR/simple_pattern.rs:15:13
   |
15 |     let x = 0 as *const i32;    
   |             ^^^^^^^^^^^^^^^

error: aborting due to 4 previous errors

